tests/types/test_resolvers.py

```diff
@@ -42,15 +42,7 @@ def name(self) -> str:
 
     assert definition.fields[0].name == "name"
     assert definition.fields[0].type == str
-
-    # We are not testing for field.base_resolver.wrapped_func == Query.name
-    # because dataclasses is deleting the attribute from the class here:
-    # https://github.com/python/cpython/blob/577d7c4e/Lib/dataclasses.py#L873-L880
-    # This prevents us from doing anything like:
-    # >>> Query().name()
-    # but we can fix this in a future release.
-
-    assert definition.fields[0].base_resolver(None) == "Name"
+    assert definition.fields[0].base_resolver(None) == Query.name(None)
```

Comment:
Shouldn't this be testing against `Query().name()`?

Reply:
yes 😊